splinepy/io/iges.py

```diff
@@ -35,4 +35,11 @@ def export(fname, splines):
     --------
     None
     """
+    # check if any spline contains volume elements
+    for ispl, spline in enumerate(splines):
+        if spline.para_dim == 3:
+            raise ValueError(
+                f"Volumetric {type(spline)} found at splines[{ispl}], which cannot be handled by the .iges file format \n"
```

Comment:
these lines are causing CIs to fail. Check [here](https://github.com/tataratat/splinepy/blob/main/CONTRIBUTING.md#automatic-formatting--style-check)

Reply:
this should be fixed by the auto-import commit

Reply:
It's not